### DISCUSSION TITLE
[Feature] low-power

### DISCUSSION THREAD

**Discussion by bigsinger** [immich-app/immich#3828] [Feature Request]
### The feature

Are there any plans to deploy the server on a low-power machine, such as a Raspberry Pi. In this way, the machine can be turned on for a long time. At present, it consumes more energy on the PC. It is troublesome to turn on the computer and start the server every time it is backed up. Is there a recommended solution?

### Platform

- [X] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
You can deploy Immich on a Pi, it just might (will) not run as smoothly as on a stronger machine. It could help to [disable the ML features](https://immich.app/docs/FAQ#why-is-immich-slow-on-low-memory-systems-like-the-raspberry-pi) or [offload them to another machine](https://immich.app/docs/guides/machine-learning).
